src/components/fields/SQFormDropdown/SQFormDropdown.tsx

```diff
@@ -162,7 +166,18 @@ function SQFormDropdown({
           })}
         </Select>
         {!isDisabled && displayHelperText && (
-          <FormHelperText>{HelperTextComponent}</FormHelperText>
+          <FormHelperText
+            sx={{
+              '&.MuiFormHelperText-root': {
+                color: 'var(--color-slate)',
+              },
+              '&.Mui-focused': {
+                color: 'var(--color-textInfoBlue)',
+              },
```

Comment:
why does the text field need the orange color added to error and dropdown does not?